src/solver/yaml_helper.jl

```diff
@@ -162,3 +162,23 @@ function job_id_from_config(config, defaults = default_config_dict())
     warn && @warn "Truncated job ID:$s may not be unique due to use of Real"
     return s
 end
+
+"""
+    run_driver(config_file)
+
+Convenience function to run the driver with a given config file.
+Useful for re-running the model with a new/modified configuration without recompiling
+"""
+function run_driver(config_file)
+    config_dict = YAML.load_file(config_file)
+    config = CA.AtmosConfig(; config_dict)
+    driver_path = joinpath(
+        dirname(@__FILE__),
+        "..",
+        "..",
+        "examples",
+        "hybrid",
+        "driver.jl",
+    )
+    include(driver_path)
```

Comment:
I believe `include` is only allowed in global scope (or, at least it's not allowed inside functions)

Reply:
Yeah, it's just running the driver file in global scope, I need to rethink this. Thanks for catching that!